### DISCUSSION TITLE
According to docs, DateTimePicker should support valueFormatter

### DISCUSSION THREAD

**Discussion by knoffi** [mantinedev/mantine#6634] [Feature requests]
Dear Mantine team,

thank you for your awesome UI library, I really enjoy it. I would like to request that the component `DateTimePicker` supports the prop `valueFormatter` (which is of type `DateFormatter`) as it is shown in the [docs](https://mantine.dev/dates/date-time-picker/?t=props.)

Thank you every much! :)

Best wishes
knoffi

P.S.: I am sorry, I could not find the sections for bugs, this is why I wrote this as a feature request.

**Comment by rtivital** [comment]
To report bugs, use GitHub Issues. I'll see what can be done about this, either prop will be removed from the docs or the functionality will be added with next patch release.

**Comment by knoffi** [comment]
Thank you very much for the fast reply.

I hope that the functionality will be added since our team could really use the valueFormatter to display icons next to date and time.

Next time, I will add it as an issue with bug-tag.
